src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -245,7 +245,7 @@ Relative path: 'SonarLint.VisualStudio.IssueVisualization.Security.dll'
 Referenced assemblies:
 - 'Microsoft.VisualStudio.Shell.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 - 'Microsoft.VisualStudio.Shell.Immutable.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
-- 'Microsoft.VisualStudio.Threading, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
+- 'Microsoft.VisualStudio.Threading, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```

Comment:
This is coming from shell 15 dependency.
I tried to manually add `Microsoft.VisualStudio.Threading` version 14 + mark version 15 with an alias, but it still doesn't work. VS complains about package downgrade and won't let me add version 14.